tests/common/test_utils_geometry.py

```diff
@@ -35,19 +38,75 @@ def test_resolve_enclosing_rbbox():
     assert all(abs(i - j) <= 1e-7 for (i, j) in zip(pred, target))
 
 
+def test_remap_boxes():
+    pred = geometry.remap_boxes(np.array([[0.5, 0.5, 0.1, 0.1, 0., 0.5]]), (10, 10), (20, 20))
+    target = np.array([[0.5, 0.5, 0.05, 0.05, 0., 0.5]])
+    assert np.all(pred == target)
+
+    pred = geometry.remap_boxes(np.array([[0.5, 0.5, 0.1, 0.1, 0., 0.5]]), (10, 10), (20, 10))
+    target = np.array([[0.5, 0.5, 0.1, 0.05, 0., 0.5]])
+    assert np.all(pred == target)
+
+    pred = geometry.remap_boxes(np.array([[0.5, 0.0, 0.5, 0.25, 0., 0.5]]), (80, 40), (160, 40))
+    target = np.array([[0.5, 0.25, 0.5, 0.125, 0., 0.5]])
+    assert np.all(pred == target)
+
+    orig_dimension = (100, 100)
+    dest_dimensions = (100, 200)
+    orig_box = np.array([[0.5, 0.5, 0.2, 0., 45, 0.5]])
+    # Unpack
+    height_o, width_o = orig_dimension
+    height_d, width_d = dest_dimensions
+    pred = geometry.remap_boxes(orig_box, orig_dimension, dest_dimensions)
+
+    x, y, w, h, a, c = orig_box[0]
```

Comment:
Can you replace "c" with "_" to avoid "unused variable warnings" please? :pray:

Reply:
ok !